build.gradle.kts

```diff
@@ -124,19 +125,19 @@ val generateResourceAttributes by tasks.registering(Exec::class) {
     "--rm",
     "-v", "$buildDir/semantic-conventions/model:/source",
     "-v", "$projectDir/buildscripts/templates:/templates",
-    "-v", "$projectDir/src/main/java/io/opentelemetry/semconv/:/output",
-    "otel/semconvgen:$generatorVersion",
-    "--only", "resource",
+    "-v", "$projectDir/src/main/java/io/opentelemetry/semconv/$semanticConventionsVersionEscaped/:/output",
+    "semconvgen:$generatorVersion",
     "--yaml-root", "/source", "code",
     "--template", "/templates/SemanticAttributes.java.j2",
-    "--output", "/output/ResourceAttributes.java",
-    "-Dclass=ResourceAttributes",
+    "--output", "/output/Attributes.java",
```

Comment:
`/optput/Attributes.java` is actually a template and will be transformed into `/output/{RootNamespace}Attributes.java` - do we want to have more flexibility and have an actual template here like (`/output/SometingElse{RootNamespace}Attributes.java`)

Reply:
resolving it for now as not substantial - it can always be done later